app/models/story_repository.rb

```diff
@@ -37,7 +37,7 @@ def active
           comments.story_id = stories.id and
           comments.created_at >= date_sub(now(), interval 3 day)
       ) as latest_comment_id')
-      .order("latest_comment_id desc")
+      .order(latest_comment_id: :desc)
```

Comment:
`/active` is slow because of this subquery. Like you noticed with `reply_count` and `depth` in #1437, this should be a materialized column for performance and simplicity. Add a column with an index and use `touch: :last_comment_at` on the association and Rails will maintain the value.

Sorry to add a database refactoring, but it'll be both a performance and code cleanup improvement.